src/js/api/config.js

```diff
@@ -39,6 +39,7 @@ const Defaults = {
         more: 'More',
         liveBroadcast: 'Live',
         loadingAd: 'Loading ad',
+        playingAd: 'Playing ad',
```

Comment:
@cmulgannon @aliciahurst 
@pajong @vseventer 

Why are we adding these messages here and not in the advertising block which already contains a list of ad strings?

We are not going to **move** these to `localization`. New advertising strings should be added to this block, which is available to ad plugins and instream.

```js
advertising: {
    cuetext: 'Advertisement',
    dynamicMessage: 'This ad will end in xx',
    podMessage: 'Ad __AD_POD_CURRENT__ of __AD_POD_LENGTH__. ',
    skipMessage: 'Skip ad in xx',
    skipText: 'Skip',
}
```

`loadingAd` and `playingAd` should go in the `advertising` block.

If we want these options to be copied from `localization`, add them all to an `advertising` block in localization.

Reply:
That's what I suggested in the last sentence of the previous comment.